acme/tests/client_test.py

```diff
@@ -821,6 +822,30 @@ def test_poll_authorizations_success(self):
             self.authz.to_json(), self.authz2.to_json(), updated_authz2.to_json())
         self.assertEqual(self.client.poll_authorizations(self.orderr, deadline), updated_orderr)
 
+    def test_query_registration_client_v2_ecdsa(self):
```

Comment:
This is one of the tests that I battled with. I think it's a good take on what we're interested in.